diff --git a/cgi-bin/var.c b/cgi-bin/var.c
index 529a40a51..661887d15 100644
--- a/cgi-bin/var.c
+++ b/cgi-bin/var.c
@@ -190,8 +190,15 @@ cgiGetCheckbox(const char *name)   /* I - Name of form field */
   const char   *value = var ? var->values[var->nvalues - 1] : NULL;
   int          ret;                    /* Return value */
 
-
-  ret = value && !_cups_strcasecmp(value, "checkbox");
+  ret = value && (
+       !_cups_strcasecmp(value, "checkbox") ||
+       !_cups_strcasecmp(value, "true") ||
+       !_cups_strcasecmp(value, "yes") ||
+       !_cups_strcasecmp(value, "on") ||
+       !_cups_strcasecmp(value, "1")
+  );
+
+  fprintf(stderr, "cgiGetCheckbox: %s=\"%d\"\n", name, ret);
 
   if (!ret && value)
   {
